airgun/helpers/base.py

```diff
@@ -0,0 +1,40 @@
+from typing import Any, Dict, List, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from airgun.entities.base import BaseEntity
+
+
+class BaseEntityHelper:
+
+    def __init__(self, entity):
+        # type: (BaseEntity) -> None
+        self._entity = entity
+
+    @property
+    def entity(self):
+        return self._entity
+
+    def read_filled_view(self, navigation_name, navigation_kwargs=None, values=None,
+                         read_widget_names=None):
+        # type: (str, Dict, Dict[str, Any], List[str]) -> Dict[str, Any]
+        """Navigate to a form with registered name navigation_name and with kwargs
```

Comment:
```
Navigate to a form using 'navigation_name' and with parameters from 'navigation_kwargs', fill the form with values and then read values for widgets from 'read_widget_names' list if supplied otherwise read all widgets values